config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
-  resources :users, only: [:index, :show]
+  resources :users, only: [:index, :show] do
+    member do
+      get 'posts', to: 'users#posts'
+    end
+    resources :posts, only: :show, controller: 'posts'
+  end
+
   resources :posts, only: [:index, :show]
```

Comment:
- [ ] Kindly add a root path, which should be the user's index action. This is required because the routes shown on the wireframe show the root path with all the users.

![root](https://github.com/TayyabKM/blog-app/assets/53124548/9f8cb7c9-f8f7-4c52-ab01-18be6bd19450)